src/unstract/sdk/utils/retry_utils.py

```
@@ -0,0 +1,306 @@
+    Returns:
+        Delay in seconds before the next retry
+    """
+    # Calculate exponential backoff
+    delay = base_delay * (multiplier**attempt)
+
+    # Cap at max delay
+    delay = min(delay, max_delay)
+
+    # Add jitter if enabled (0-25% of delay)
+    if jitter:
+        jitter_amount = delay * random.uniform(0, 0.25)
```

Ruff (0.13.1)

Standard pseudo-random generators are not suitable for cryptographic purposes

(S311)